README.md

```diff
@@ -23,9 +23,10 @@ Repository to describe, develop, document and test the QualityOnDemand API famil
 * Schedule: bi-weekly, Friday, 2 PM CET/CEST (13:00 UTC, 12:00 UTC during European DST). For date/time of next meeting see previous [meeting minutes](https://github.com/camaraproject/QualityOnDemand/tree/main/documentation/MeetingMinutes).
 * Meeting link: <a href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_MGM3YTBjYWYtNGVmNy00Mjk1LWJhMTktZjI1M2NjNzg2ZDFh%40thread.v2/0?context=%7b%22Tid%22%3a%22bde4dffc-4b60-4cf6-8b04-a5eeb25f5c4f%22%2c%22Oid%22%3a%2237ff36be-0e4d-42c3-ac06-7b904f0f6b24%22%7d">Microsoft Teams Meeting</a> 
 
-## Results and Status
-* The latest release and version of Quality-On-Demand (QoD) API is 0.8.1. This is the seconed alpha version of the API. There are bug fixes to be expected and incompatible changes in upcoming versions. It is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
-* Version 0.8.1 of the API :
+## Status and released versions
+* Note: Please be aware that the project will have frequent updates to the **main** branch. **There are no compatibility guarantees associated with code in any branch, including main, until it has been released.** For example, changes may be reverted before a release is published. **For best results, use the latest published release**.
```

Comment:
I think, we should re-phrase "...until it has been released" to ""...until a new intermedia release is created". 
"For example, changes may be reverted before a release is **published**." --> For example, changes may be reverted before a release is **created**.
"**For best results, use the latest published release**." --> **For best results, use the latest available release**.